### PR TITLE
Update nginx configurations to handle proxy with no ssl endpoint

### PR DESCRIPTION
**Pull request by pgraverdy**
Updates configurations to  support 1 and 2 vms deployments
- add a setup for a frontend with certbot/nginx that redirects requests to another vm
- add a setup to have nginx only redirect http requests to the app's local containers

### CODE REVIEW

File: docker/configs/nginx/front-ssl-proxy.conf [hunk 1]
```diff
@@ -1,35 +1,30 @@
...
+        server APP_IP_ADDR;
 }
 
 server {
     listen 80;
     server_name WEB_HOSTNAME;
     server_tokens off;
 
-    # for certbot cert setup/renewal dns challenge
+    # for certbot cert setup/renewal dns challend
```
Comment (rprimet) [inline docker/configs/nginx/front-ssl-proxy.conf:19]: challenge?

File: docker/configs/nginx/front-ssl-proxy.conf [hunk 2]
```diff
@@ -1,35 +1,30 @@
 #
 # Nginx configuration for docker-compose nginx container
 #
+# This configuration assumes an nginx container launched on o different host/VM
```
Comment (VincentRouvreau) [inline docker/configs/nginx/front-ssl-proxy.conf:4]:
```suggestion
# This configuration assumes an nginx container launched on a different host/VM
```

File: docker/configs/nginx/front-ssl-proxy.conf (hunk 1 above)
Comment (VincentRouvreau) [reply docker/configs/nginx/front-ssl-proxy.conf:19]:
```suggestion
    # for certbot cert setup/renewal dns challenge
```

File: docker/configs/nginx/front-ssl-proxy.conf [hunk 3]
```diff
@@ -1,35 +1,30 @@
...
+# certificates renewal.
+#
 
-upstream app-server {
-        server icubam_www_server:8888;
-}
-
-upstream backoffice-server {
-        server icubam_bo_server:8890;
+upstream app-proxy {
```
Comment (rprimet) [inline docker/configs/nginx/front-ssl-proxy.conf:10]: `app-proxy`  -> app_origin ?

File: docker/configs/nginx/front-ssl-proxy.conf [hunk 4]
```diff
@@ -1,13 +1,14 @@
...
```
Comment (pgraverdy) [inline docker/configs/nginx/front-ssl-proxy.conf:10]:
```suggestion
upstream app-origin {
```

File: docker/configs/nginx/front-ssl-proxy.conf [hunk 5]
```diff
@@ -42,19 +37,12 @@ server {
...
-        proxy_pass http://backoffice-server;
-        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
-        proxy_set_header X-Forwarded-Proto $scheme;
-    }
-
     location / {
         proxy_set_header Host $host;
         proxy_set_header X-Real-Ip $remote_addr;
-        proxy_pass http://app-server;
+        proxy_pass http://app-proxy;
```
Comment (pgraverdy) [inline docker/configs/nginx/front-ssl-proxy.conf:43]:
```suggestion
        proxy_pass http://app-origin;
```

**Pull request merged**